jaxrs-clients/src/main/java/com/palantir/remoting1/jaxrs/FeignJaxRsClientBuilder.java

```diff
@@ -163,7 +163,7 @@ private Decoder createDecoder(ObjectMapper objectMapper) {
                 .randomGenerator(new Random())
                 .state(new ThreadLocalServerClientAndLocalSpanState(
                         getIpAddress(), 0 /** Client TCP port. */, userAgent))
-                .spanCollector(new SlfLoggingSpanCollector("ClientTracer(" + userAgent + ")"))
+                .spanCollector(new SlfLoggingSpanCollector("tracing.client." + userAgent))
```

Comment:
How do you adjust the log config for the `ClientTracer(com.palantir.product` form? For `tracing.client.com.palantir.product` you can set it as `tracing: INFO` and send it to the desired appender.